### PR TITLE
standardnotes: 3.8.18 -> 3.9.5

### PR DESCRIPTION
**Pull request by chuangzhu**
###### Motivation for this change

Bump Standard Notes.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] i686-linux
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by chuangzhu** [comment]
Result of `nixpkgs-review pr 148518` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>standardnotes</li>
  </ul>
</details>

File: pkgs/applications/editors/standardnotes/default.nix [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
+    i686-linux = "i386";
     x86_64-linux = "x86_64";
   }.${stdenv.hostPlatform.system};
 
   sha256 = {
-    i386-linux = "1xiypsmvpk8i6kab862pipbdfb0y5d5355hdwjmva7v7g26aa7h7";
-    x86_64-linux = "03qlxlgyypnvcr40jh6i4wriyax2jbfhrb798cq0n7qlc1y4pg8r";
+    i686-linux = "sha256-7Mo8ELFV6roZ3IYWBtB2rRDAzJrq4ht9f1v6uohsauw=";
+    x86_64-linux = "sha256-9VPYII9E8E3yL7UuU0+GmaK3qxWX4bwfACDl7F7sngo=";
   }.${stdenv.hostPlatform.system};
```
Comment (SuperSandro2000) [inline pkgs/applications/editors/standardnotes/default.nix:17]: Those should have a or throw unsupported system to eval also on other platforms.

Comment (chuangzhu) [reply pkgs/applications/editors/standardnotes/default.nix:17]: Should I revert it to “i386”?

Comment (bobby285271) [reply pkgs/applications/editors/standardnotes/default.nix:17]: If you are not sure why and how to add the throw, maybe just check https://github.com/NixOS/nixpkgs/pull/143647#discussion_r739261918.

Comment (bobby285271) [reply pkgs/applications/editors/standardnotes/default.nix:17]: Keeping i686-linux should be fine I guess.

**Pull request merged**